api/docs/v2/storefront/index.yaml

```diff
@@ -24,15 +26,44 @@ paths:
       security:
         - orderToken: []
       requestBody:
+        required: true
         content:
           application/json:
             schema:
               type: object
               properties:
                 variant_id:
-                  type: string
+                  type: integer
+                  example: 10
                 quantity:
-                  type: number
+                  type: integer
+                  example: 1
+  /products/{id}:
+    get:
+      description: "To view the details for a single product, make a request using that product's permalink:<br /> <code>GET /api/v2/products/a-product</code> <br />You may also query by the product's id attribute: <code>GET /api/v1/products/1
+                    </code> <br /> Note that the API will attempt a permalink lookup before an ID lookup."
+      tags:
+      - Products
+      operationId: show
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: integer
```

Comment:
in my opinion, `id` should be accepting strings in case someone uses something like uuid for that